crates/sui-graphql-rpc/src/extensions/query_limits_checker.rs

```diff
@@ -287,11 +357,21 @@ impl QueryLimitsChecker {
             }
             level_len = que.len();
         }
+
+        if cost.output_nodes < 1 {
```

Comment:
Do we have negative number of output nodes? Not sure I get this condition.

Reply:
Ah this might've been something left over from my initial approach. Will test to see if I can remove this